tools/backups2datalad.py

```diff
@@ -736,5 +754,59 @@ def pdb_excepthook(exc_type, exc_value, tb):
         pdb.post_mortem(tb)
 
 
+def mkrelease(repo: Path, dandiset: RemoteDandiset, commitish: str = None) -> None:
+    # `dandiset` must have its version set to the published version
+    if commitish is None:
+        asset_ids = {a.identifier for a in dandiset.get_assets()}
+        repo_assets = json.loads(
+            (repo / ".dandi" / "assets.json").read_text(encoding="utf-8")
+        )
+        if asset_ids == {a["asset_id"] for a in repo_assets}:
```

Comment:
let's move this logic into a helper `compare_assets(assets, repo_assets)` or alike. I think that comparing by `assets_ids` should be fine, but if later we discover that it is not sufficient, we better fix it in a single location (now the same comparison done here and below while traversing the history)

Reply:
Helper function added.